olm_deploy/operatorregistry/registry-deployment.yaml

```diff
@@ -9,6 +9,8 @@ spec:
       registry.operator.cluster-logging: "true"
   template:
     metadata:
+      annotations:
+        workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
```

Comment:
This deployment is only used during testing.  Do we still need the change here?

Reply:
No, in that case it probably doesn't matter.  I was perhaps overly exuberant in my grepping :)

Reply:
Would you like me to remove this one?